msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting.md

```diff
@@ -0,0 +1,205 @@
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error, and result. The *error* can either contain an error of type *SdkError*, in case of an error, or null when share is successful. The *result* can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval.|
+
+### Example
+
+```javascript
+microsoftTeams.meeting.getAppContentStageSharingState((err, result) => {
+    if (result.isAppSharing) {
+        // Indicates app has permission to share contents to meeting stage.
+    }
+});
+```
+
+The JSON response body for the `getAppContentStageSharingState` API is:
+
+```json
+{
+   "isAppSharing":true
+} 
+```
+
+### Response codes
+
+The following table provides the response codes:
+
+|Response code|Description|
+|---|---|
+| **500** | Internal error. |
+| **501** | API isn't supported in the current context.|
+| **1000** | App doesn't have proper permissions to allow share to stage.|
+
+## Get app content stage sharing capabilities API
+
+The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage.
+
+### Query parameter
+
+The following table includes the query parameter:
+
+|Value|Type|Required|Description|
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error, and result. The *error* can either contain an error of type *SdkError*, or null when share is successful. The result can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval.|
+
+### Example
+
+```javascript
+microsoftTeams.meeting.getAppContentStageSharingCapabilities((err, result) => {
+    if (result.doesAppHaveSharePermission) {
+        // Indicates app has permission to share contents to meeting stage.
+    }
+});
+```
+
+The JSON response body for `getAppContentStageSharingCapabilities` API is:
+
+```json
+{
+   "doesAppHaveSharePermission":true
+} 
+```
+
+### Response codes
+
+The following table provides the response codes:
+
+|Response code|Description|
+|---|---|
+| **500** | Internal error. |
+| **1000** | App doesn't have permissions to allow share to stage.|
+
+## Code sample
+
+|Sample name | Description | C# | Node.js |
+|----------------|-----------------|--------------|----------------|
+|Meeting stage sample | Sample app to show a tab in meeting stage for collaboration | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/nodejs) |
+| In-meeting notification | Demonstrates how to implement in-meeting notification using bot. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/nodejs) |
```

Comment:
Called out in the other docs too, but this might make sense to include here. Especially since we use the share app content to stage API.

| In-Meeting Document Signing | Demonstrates how to implement a document signing Teams app. Includes sharing specific app content to stage, Teams SSO and user specific stage view. | View | |